src/app/core/filter/filter/filter.component.ts

```diff
@@ -102,9 +102,12 @@ export class FilterComponent<T extends Entity = Entity> implements OnChanges {
 
   filterOptionSelected(filter: Filter<T>, selectedOptions: string[]) {
     filter.selectedOptionValues = selectedOptions;
-    this.hasActiveFilters = this.filterSelections.some(
-      (f) => f.selectedOptionValues?.length > 0,
-    );
+    setTimeout(() => {
```

Comment:
I guess this had change detection issues? Any idea why/what exactly? (we should probably add code comments to such workarounds for documentation)